content/geodesic/module/_index.md

```diff
@@ -5,3 +5,49 @@ description: "Geodesic modules are Git repositories that extend the Geodesic bas
+- [Terraform Modules](/terraform-modules/)
+- [Polyrepos]({{< relref "glossary/polyrepo.md" >}}) to logically organize infrastructure. Use modules to import logic.
+
+We prefer the polyrepo approach to the monorepo approach when it comes in infrastructure. Git repositories allow us to surgically tag/version parts of infrastructure.
+
+For example, we often prescribe organizing repositories by AWS organization. For the purpose of documentation, we'll use `cloudposse.co` as the example organization.
+
+# Reference Architectures
+
+## root.cloudposse.co
+
+The [`root.cloudposse.co`](https://github.com/cloudposse/root.cloudposse.co) module represents an organization's "root" or "top level" AWS account. This is typically your billing organization and handles IAM user account provisioning.
+
+It should also use the `terraform` resource  [`aws_organizations_organization`](https://www.terraform.io/docs/providers/aws/r/organizations_organization.html) to provision all the subaccounts below.
+
+## prod.cloudposse.co
+
+The [`prod.cloudposse.co`](https://github.com/cloudposse/prod.cloudposse.co) module represents an organization's "production infrastructure". It's a subaccount (organization) of the [`root.cloudposse.co`](https://github.com/cloudposse/root.cloudposse.co).
+
+## staging.cloudposse.co
+
+The [`staging.cloudposse.co`](https://github.com/cloudposse/staging.cloudposse.co) module represents an organization's "staging infrastructure".  Typically, we treat this as a "pre-production" environment, which means it should remain stable and treated as though it were a real production environment. This environment gives developers access to a "production-like" environment, without the consequences, risks and liaiblities of actually operating in production.
+
+## dev.cloudposse.co
+
+The [`dev.cloudposse.co`](https://github.com/cloudposse/dev.cloudposse.co) module represents an organization's "development infrastructure". This module is used as a sandbox environment where developers and test the waters and get familiar with AWS. We prescribe that organizations give all developers "Administrator" level privileges to this account in order to give them an unfettered opporutnity at developing infrastructure.
+
+## audit.cloudposse.co
+
+The [`audit.cloudposse.co`](https://github.com/cloudposse/audit.cloudposse.co) module represents an organization's "audit logs infrastructure".  By leveraging our [`terraform-aws-cloudtrail`](https://github.com/cloudposse/terraform-aws-cloudtrail) and [`terraform-aws-cloudtrail-s3-bucket`](https://github.com/cloudposse/terraform-aws-cloudtrail-s3-bucket)  modules, we make it exceptionally easy to ship CloudTrail logs from all your organizations to one central, highly restrictive organization whose sole purpose is to provide a tamper-proof, encrypted, system of record for infrastructure logs with [lifecycle rules]({{< relref "terraform/tips-tricks.md#s3-bucket-lifecycle-rules" >}}).
```

Comment:
> from all your organizations

From all your accounts in the organization?
Or from Organizational Units (OUs)?